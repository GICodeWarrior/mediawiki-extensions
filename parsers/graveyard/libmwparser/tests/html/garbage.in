<table>garbage<tbody>garbage<tr>garbage<td>cell</td>garbage</tr>garbage<tr>garbage<td>cell</td>garbage</tr>garbage</tbody>garbage</table>

<dl>garbage<dd>dd</dd>garbage<dt>dd</dt>garbage<dd>dd</dd>garbage</dl>

