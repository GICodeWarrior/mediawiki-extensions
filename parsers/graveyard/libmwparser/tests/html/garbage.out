BEGIN ARTICLE
    BEGIN GARBAGE
        WORD[garbage]
    END GARBAGE
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN GARBAGE
                WORD[garbage]
            END GARBAGE
            BEGIN GARBAGE
                WORD[garbage]
            END GARBAGE
            BEGIN ROW
                BEGIN CELL
                    WORD[cell]
                END CELL
                BEGIN GARBAGE
                    WORD[garbage]
                END GARBAGE
            END ROW
            BEGIN GARBAGE
                WORD[garbage]
            END GARBAGE
            BEGIN GARBAGE
                WORD[garbage]
            END GARBAGE
            BEGIN ROW
                BEGIN CELL
                    WORD[cell]
                END CELL
                BEGIN GARBAGE
                    WORD[garbage]
                END GARBAGE
            END ROW
            BEGIN GARBAGE
                WORD[garbage]
            END GARBAGE
            BEGIN GARBAGE
                WORD[garbage]
            END GARBAGE
        END TABLE BODY
    END TABLE
    BEGIN GARBAGE
        WORD[garbage]
    END GARBAGE
    BEGIN DEFINITION LIST
        BEGIN DEFINITION ITEM
            WORD[dd]
        END DEFINITION ITEM
        BEGIN GARBAGE
            WORD[garbage]
        END GARBAGE
        BEGIN DEFINED TERM ITEM
            WORD[dd]
        END DEFINED TERM ITEM
        BEGIN GARBAGE
            WORD[garbage]
        END GARBAGE
        BEGIN DEFINITION ITEM
            WORD[dd]
        END DEFINITION ITEM
        BEGIN GARBAGE
            WORD[garbage]
        END GARBAGE
    END DEFINITION LIST
END ARTICLE
